src/components/Button.js

```diff
@@ -2,15 +2,27 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = (props) => {
-  const { name } = props;
+  const { name, color, width } = props;
   return (
-    <button type="button">
+    <button
+      type="button"
+      className={[
+        (color) ? 'white' : undefined,
+        (width) ? 'double-size' : undefined,
+      ].join(' ')}
+    >
       {name}
     </button>
   );
 };
 Button.propTypes = {
   name: PropTypes.string.isRequired,
+  color: PropTypes.bool,
+  width: PropTypes.bool.isRequired,
+};
+
+Button.defaultProps = {
+  color: false,
```

Comment:
Hi, lets write out code in a way that this default will be 'orange' string, that way we don't need to set undefined, the default will kick in.